src/org/webonise/perimetercalculator/Application.java

```diff
@@ -0,0 +1,32 @@
+package org.webonise.perimetercalculator;
+
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class Application {
+    private static Logger LOGGER = Logger.getLogger(Application.class.getName());
+    private LoggerManager loggerManager = new LoggerManager();
+    private static final int UPPERBOUND = 3;
+    private static final int LOWERBOUND = 8;
+    private int noOfSides;
+    private double sideLength;
+
+    public void startApplication() {
+        Scanner input = new Scanner(System.in);
+        LOGGER = loggerManager.getLOGGER(LOGGER);
+        LOGGER.log(Level.FINE, "Enter No of Sides and Length of Side");
+        noOfSides = input.nextInt();
+        sideLength = input.nextDouble();
+        if (noOfSides >= UPPERBOUND && noOfSides <= LOWERBOUND) {
```

Comment:
We can extract these conditions inside another method. Like `validateNoOfSites()`